src/components/layers/basemaps/BasemapCard.js

```diff
@@ -75,61 +22,29 @@ const BasemapCard = props => {
         toggleBasemapExpand,
         toggleBasemapVisibility,
         changeBasemapOpacity,
-        classes,
     } = props;
 
     return (
-        <Card className={classes.card} data-test="basemapcard">
-            <CardHeader
-                classes={{
-                    root: classes.header,
-                    title: classes.title,
-                    subheader: classes.subheader,
-                }}
-                title={i18n.t(name)}
-                subheader={subtitle}
-                action={
-                    <Tooltip
-                        title={
-                            isExpanded ? i18n.t('Collapse') : i18n.t('Expand')
-                        }
-                        classes={{
-                            tooltipPlacementBottom: classes.tooltip,
-                        }}
-                    >
-                        <IconButton
-                            className={classes.expand}
-                            onClick={toggleBasemapExpand}
-                            style={{ backgroundColor: 'transparent' }}
-                        >
-                            {isExpanded ? <CollapseIcon /> : <ExpandIcon />}
-                        </IconButton>
-                    </Tooltip>
-                }
+        <LayerCard
+            title={i18n.t(name)}
```

Comment:
Does this translate correctly?

Reply:
I removed `i18n.t(...)`, as translations are already handled in constants/basemaps.js and in the Maintenance app (external basemaps).